<launch>

  <arg name="nodelet_manager" default="lidar_start_nodelet_manager"/>
  <node pkg="nodelet"  type="nodelet" name="$(arg nodelet_manager)" args="manager" output="screen"/>

  <node pkg="nodelet" type="nodelet" name="LidarStartNodelet" args="load lidar_start/LidarStartNodelet $(arg nodelet_manager)"  output="screen">
    <!-- LiDAR params -->
    <param name="lidar_type" value="VLP16"/> <!-- P40P RSL32 VLP16 is now avaiable -->
    <param name="ip" value="195.0.0.201"/>
    <param name="data_port" value="2368"/>
    <param name="mode" value="0"/>
    <!-- Topic Setting -->
    <param name="frame_id" value="velodyne"/>
    <param name="origin_cloud_topic" value="velodyne_points"/>
  </node>

</launch>
